core/include/gnuradio-4.0/BlockRegistry.hpp

```diff
@@ -82,6 +92,25 @@ class BlockRegistry {
         return _block_types;
     }
 
+    bool
+    isBlockKnown(std::string_view block) const {
+        return _block_type_handlers.find(std::string(block)) != _block_type_handlers.end();
+    }
+
+    auto
+    knownBlockParametrizations(std::string_view block) const {
```

Comment:
spelling: 'parameterisation' or 'parameterization' ... otherwise, we are para-metricizing the blocks. ;-)